README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+helper because it has code that shows green or red based on if rating is 10 or 1.
 ```
```

Comment:
Helper method is the correct answer, but that's because they used to contain logic that determines what will be shown in a given view.